backend/audit/intakelib/common/error_messages.py

```diff
@@ -1,5 +1,6 @@
 messages = {
     "check_uei_exists": "You did not provide a UEI on the coversheet",
+    "check_uei_schema": "The auditee UEI is not valid",
```

Comment:
Actually, *technically*:

1. Checking against the schema tells us whether it is correctly formed or not.
2. Checking against the value in the checklist tells us whether or not the UEI matches the value in that the auditor/auditee used to create the audit.

Does this code cover both cases? Or, is this only doing the "correct shape" check? 

If so, should we handle the case that the user encountered, which was that they were using a correctly formed UEI, but it didn't match?

(I may not be understanding the code completely.)

Reply:
And, finishing my thread...

I checked the ticket, and it seems to be different than the one I am thinking of. I know we had a ticket recently where the user entered a well-formed UEI, but it was for an audit different than the one they were submitting. It passed validation on upload, but was caught at cross-val as incorrect. They asked/suggested we check that the UEI is correct at time of workbook upload.

If that's easy to add to this ticket, perhaps we should. It could be a separate, simple UEI check applied to all workbooks *after* the schema validation check. 

Happy to chat if none of this rambling is clear. :coffee: ...

Reply:
@jadudm The current changes in this PR should do what you described.